src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/hmi/on_tts_language_change_notification.cc

```diff
@@ -59,20 +59,26 @@ OnTTSLanguageChangeNotification::~OnTTSLanguageChangeNotification() {}
 void OnTTSLanguageChangeNotification::Run() {
   LOG4CXX_AUTO_TRACE(logger_);
 
-  HMICapabilities& hmi_capabilities = hmi_capabilities_;
-
-  hmi_capabilities.set_active_tts_language(
+  hmi_capabilities_.set_active_tts_language(
       static_cast<hmi_apis::Common_Language::eType>(
           (*message_)[strings::msg_params][strings::language].asInt()));
 
   /* need to clarify, because unchanged VR
   cause WRONG_LANGUAGE on Register */
-  hmi_capabilities.set_active_vr_language(
+  hmi_capabilities_.set_active_vr_language(
       static_cast<hmi_apis::Common_Language::eType>(
           (*message_)[strings::msg_params][strings::language].asInt()));
 
+  std::vector<std::string> sections_to_update;
```

Comment:
@ydementieiev `std::vector<std::string> sections_to_update{hmi_response::language};` instead of 2 lines? If you are not extending it afterwards

Reply:
@AByzhynar I changed it in all needed [place](https://github.com/LuxoftSDL/sdl_core/pull/35/commits/cdab57c9b89acd94155de176f4a223e55f72602a)